Sources/Kitura/RouterResponse.swift

```diff
@@ -179,8 +179,71 @@ public class RouterResponse {
         }
     }
 
-    // MARK: End
+    /// Add a cookie to the response.
+    ///
+    /// This function creates an `HTTPCookie`  from the provided attributes and adds it to the `cookies` dictionary.
+    /// - Parameter name: The cookie’s name.
+    /// - Parameter value: The cookie‘s value.
+    /// - Parameter domain: The domain of the cookie.
+    /// - Parameter path: The cookie’s path.
+    /// - Parameter otherAttributes: An array of  any other optional cookie attributes
+    public func addCookie(name: String, value: String, domain: String, path: String, otherAttributes: [OptionalCookieAttribute]? = nil ) {
+        var cookieProperties = [HTTPCookiePropertyKey: Any]()
+        cookieProperties[HTTPCookiePropertyKey.name] = name
+        cookieProperties[HTTPCookiePropertyKey.value] = value
+        cookieProperties[HTTPCookiePropertyKey.domain] = domain
+        cookieProperties[HTTPCookiePropertyKey.path] = path
+        if let otherAttributes = otherAttributes, otherAttributes.isEmpty == false {
+            for attribute in otherAttributes {
+                switch attribute._value {
+                case .portList(let ports):
+                    if let ports = ports {
+                        cookieProperties[HTTPCookiePropertyKey.port] = ports
+                    }
+
+                case .expires(let expiresDate):
+                    if let expiresDate = expiresDate {
+                        cookieProperties[HTTPCookiePropertyKey.expires] = expiresDate
+                    }
+
+                case .maximumAge(let maxAge):
+                    cookieProperties[HTTPCookiePropertyKey.maximumAge] = maxAge
+
+                case .originURL(let originURL):
+                    cookieProperties[HTTPCookiePropertyKey.originURL] = originURL
+
+                case .version(let cookieVersion):
+                    cookieProperties[HTTPCookiePropertyKey.version] = cookieVersion
+
+                case .discard(let discard):
+                    cookieProperties[HTTPCookiePropertyKey.discard] = discard
+
+                case .isSecure(let secure):
+                    cookieProperties[HTTPCookiePropertyKey.secure] = secure ? "YES" : "NO"
+
+                case .comment(let comment):
+                    if let comment = comment {
+                        cookieProperties[HTTPCookiePropertyKey.comment] = comment
+                    }
+
+                case .commentURL(let commentURL):
+                    if let commentURL = commentURL {
+                        cookieProperties[HTTPCookiePropertyKey.commentURL] = commentURL
+                    }
+
+                case.custom(let key, let value):
```

Comment:
Now that we're free to extend the struct at any time, do we need a `.custom`?  If not, it would be cleaner to remove it.
Otherwise, can you add the missing space between `case` and `.custom` here.